tracker/labels.go

```diff
@@ -87,7 +87,8 @@ func (t *myTracker) RenameFirstTime(det *track) *track {
 		t.classCounter[baseLabel] = classCount + 1
 	}
 	countLabel := baseLabel + "_" + strconv.Itoa(t.classCounter[baseLabel])
-	label := countLabel + "_" + GetTimestamp()
+	classLabel := det.detClassification.Label()
```

Comment:
If detClassification is nil, this will panic. You need to check for the existence of the classification field and put in a default empty string or something if classification if not set.